TestingLowerBounds/Divergences/StatInfo.lean

```diff
@@ -147,7 +151,9 @@ lemma toReal_statInfo_eq_min_sub_integral (μ ν : Measure 𝒳) [IsFiniteMeasur
   rw [toReal_bayesBinaryRisk_eq_integral_min,
     MonotoneOn.map_min (fun _ _ _ hb hab ↦ ENNReal.toReal_mono hb hab) hμ hν]
 
-lemma toReal_statInfo_eq_min_sub_integral' {μ ν ζ : Measure 𝒳} [IsFiniteMeasure μ] [IsFiniteMeasure ν]
+#check Measure.rnDeriv_eq_div'
+
```

Comment:
```suggestion
```
Please scroll through the changes of your PRs before opening them. You often leave things like this or stray examples or comments.